packages/core/admin/server/services/role.js

```diff
@@ -354,6 +363,11 @@ const assignPermissions = async (roleId, permissions = []) => {
     await getService('metrics').sendDidUpdateRolePermissions();
   }
 
+  strapi.eventHub.emit(
+    'role.permissions',
```

Comment:
I think `role.permissions` isn't really clear, instead we should probably emit events like `permission.create` and `permission.delete`, to be consistent with other types of events. And the best place to do it would be at the permission service level ([probably there](https://github.com/strapi/strapi/blob/d26434b1c92879468d950cbc08dc27aa0fc151d1/packages/core/admin/server/services/permission/queries.js))